BootloaderCorePkg/Tools/SblFwUpdate.py

```diff
@@ -0,0 +1,32 @@
+## @ SblFwUpdate.py
+#
+# Copyright (c) 2017-2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+import os
+import re
+import sys
+import shutil
+
+def Usage():
+    print "SblFwUpdate"
+    print "Usage:"
+    print "    SblFwUpdate  <Input Capsule File>"

```

Comment:
we should use py3 style print.  Using print ("...") instead.

Reply:
Done